src/ComponentDetailsPage.jsx

```diff
@@ -19,6 +23,36 @@ const metadata = data.metadata;
 const tags = Object.keys(metadata.tags || {});
 const detailsUrl = `#/${REPL_ACCOUNT}/widget/ComponentDetailsPage?src=${src}`;
 const shareUrl = `https://${REPL_NEAR_URL}${detailsUrl}`;
+const accountProfileDescription =
+  Social.getr(`${accountId}/profile`).description ?? "";
+console.log(accountProfileDescription, "desc");
+const descMaxWords = 15;
+const componentDescMaxWords = 25;
+if (accountProfileDescription) {
+  const text = accountProfileDescription.split(" ");
+  accountProfileDescription = text.slice(0, descMaxWords);
+  if (text.length >= descMaxWords) {
+    accountProfileDescription.push("...");
+  }
+  accountProfileDescription = accountProfileDescription.join(" ");
+}
+
+const stats = {
+  developerStats: {
+    numberOfComponentsPublished: 40,
+    developerSince: new Date("2015-01-01").getFullYear(),
+  },
+  componentStats: {
+    numberOfForks: 10,
+    impressions: 93,
+    parents: [
+      "mob.near/widget/Image",
+      "mob.near/widget/Image",
+      "mob.near/widget/Image",
+      "mob.near/widget/Image",
+    ],
+  },
+};
```

Comment:
Why is this data hardcoded?

Reply:
Should have commented on this review before. 

This is a final API endpoint we are waiting on, but I wanted to get a general review of this PR before that.

Reply:
Sounds good! Maybe in the future, you could mark your PR as a draft if there's still pieces you need to finalize.